src/plugin.js

```diff
@@ -35,6 +35,7 @@ import OperatorStatusTelemetry from './providers/user/operator-status-telemetry.
 import LatestTelemetryProvider from './providers/latest-telemetry-provider.js';
 import PollQuestionParameter from './providers/user/poll-question-parameter.js';
 import PollQuestionTelemetry from './providers/user/poll-question-telemetry.js';
+import ExportToCSVAction from "./actions/exportToCSV/plugin";
```

Comment:
The import for the plugin has the same name as the Plugin class itself. Might cause confusion?

Reply:
Done